.gitignore

```diff
@@ -29,7 +29,7 @@
 
 /public/assets
 .byebug_history
-
+file1.env
```

Comment:
If you're using a `.env` file to hide your credentials, you need to make sure you don't commit the file to your git history! Otherwise, there's no point in using `dotenv`, since people will be able to see your credentials anyway.